### DISCUSSION TITLE
Should a new investor use this tool? If Yes, what would be the best way to go forward with?

### DISCUSSION THREAD

**Discussion by dummifiedme** [pranjal-joshi/Screeni-py#106] [Q&A] [answered]
Also, if there is something you would like to suggest for better understanding of the stocks or the parameters used here, it would be really appreciated.

**Comment by pranjal-joshi** [comment, marked as answer]
Hello @dummifiedme and Welcome to the Screeni-py community!

### Question 1:

> Should a new investor use this tool? If Yes, what would be the best way to go forward with?

Definitely, but with certain rules! If you are just getting started your trading journey, I hope screenipy will surely help you to get on-board easily.
With this tool, you can primarily focus on **Momentum Trading** which means catching the breakouts just in time, getting some really good gains, and hopping to the next breakout when your existing positions show the signs of trend change!

But before blindly trusting the screened results, Try to follow this checklist to get started!
- [x] Always Verify Breakout/Resistance values suggested by Screenipy with Charts 📈 
- [x] Always know when to exit (Especially, Set your stop-loss, because no one can get 10/10 trades correct!)
- [x] Try to find the setups which work for you. It will be your edge in the market. For example, Someone can trade bullish breakouts in a better way while the other person can trade more confidently in a sideways market. Find your style, get an Edge!
- [x] Observe, Analyze and Respect the behavior of the market. We all pay SLs sometimes :wink:, so don't take it personally that the market is against you!

### Question 2:

> what would be the best way to go forward with?

Please join this **[Discussion](https://github.com/pranjal-joshi/Screeni-py/discussions/95)**. Some of the profitable setups are shared here for beginners to get started.
If you find something that works for you consistently, Feel free to post the setup in the same discussion! Contribute back to the Open-Source with your Edge! 💯 

### Question 3:

> Also, if there is something you would like to suggest for a better understanding of the stocks or the parameters used here

Sure. We have documented a few features **[Here](https://github.com/pranjal-joshi/Screeni-py#understanding-the-result-table)**
Though, many of the hidden features are not yet properly documented, Feel free to explore the software itself for it.
Also, if you want to contribute some new documentation, Please feel free to follow the [Guidelines for Contributing](https://github.com/pranjal-joshi/Screeni-py/blob/new-features/CONTRIBUTING.md)

In case of any queries, feel free to continue this discussion! 😄

**Reply by dummifiedme** [reply]
Thanks for the reply. As I was trying it out, I couldn”t get it to work. It seemed to throw an error “No network” after seleting the options.

PS. I am on Manjaro linux.

**Reply by pranjal-joshi** [reply]
Please open an independent issue [here](https://github.com/pranjal-joshi/Screeni-py/issues) for technical glitches.
Providing error logs or screenshots will be helpful along with the issue.

**Reply by dummifiedme** [reply]
I raised an issue with a screenshot #109. I don't know how to provide error log. 


Also, is there a portal or a guide where I can learn "How to"? (not necessarily related to the code itself, but rather in general.)

eg. What is breakout or consolidation? When to do what? etc.
